deploy.sh

```diff
@@ -95,10 +108,17 @@ server {
         proxy_set_header Upgrade \$http_upgrade;
         proxy_set_header Connection \"upgrade\";
 
-        proxy_pass  http://$(whoami);
+        proxy_pass  http://$USER;
         proxy_redirect off;
     }
-}" > /etc/nginx/conf.d/$(whoami).conf
+}" > /etc/nginx/conf.d/$USER.conf
+
+# Save current config
```

Comment:
So that, when an instance has been deployed once, we just need to run `./deploy.sh` without extra argument.
This would be necessary for prod: as we deploy it often, we don't want to have to specify the ports and public host each time.